src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/TestWebApi.cs

```diff
@@ -41,8 +41,7 @@ public TestWebApi(TestContext context, IHook<object> eventMessageHook, IHook<ICo
                     { "ConfigNames", "SFA.DAS.EmployerIncentives" }
                 };
 
-            _paymentProfiles = new IncentivePaymentProfileListBuilder().Build();
-
+            _paymentProfiles = new IncentivePaymentProfilesService().Get().GetAwaiter().GetResult();
```

Comment:
dis doesn't look good

Reply:
can't we just inject the result set ready for action?

Reply:
Looks like _paymentProfiles wasn't being used anyway